content/en/account_management/audit_trail_events.md

```diff
@@ -174,6 +175,12 @@ See the [Audit Trail documentation][2] for more information on setting up and co
 | Synthetic variable | A user created, modified, or deleted a synthetic variable. | `@evt.name:"Synthetics Monitoring" @asset.type:synthetics_variable` |
 | Synthetic settings | A user modified synthetic settings (quotas, PL access) and the previous and new setting values. | `@evt.name:"Synthetics Monitoring" @asset.type:synthetics_settings @action:modified` |
 
+### Reference Table events
+| Name                     | Description of audit event                                          | Query in audit explorer                           |
+| ------------------------ | ------------------------------------------------------------------- | --------------------------------------------------|
+| Reference Table | A user created, deleted, or modified a reference table. | `@evt.name:"Reference Tables" @asset.type:reference_table @action:(created OR deleted OR modified)` |
+| Reference Table File | A user uploaded a file or imported a file via a cloud provider for a reference table. | `@evt.name:"Reference Tables" @asset.type:reference_table_file @action:(uploaded OR imported)` |
```

Comment:
```suggestion
| Reference Table File | A user uploaded a file or imported a file with a cloud provider for a reference table. | `@evt.name:"Reference Tables" @asset.type:reference_table_file @action:(uploaded OR imported)` |
```
Ah, sorry, one more nit that I hadn't noticed. I'll go ahead and commit this one.